libtsuba/src/RDG.cpp

```diff
@@ -43,7 +43,8 @@ namespace {
 // special partition property names
 const char* kMirrorNodesPropName = "mirror_nodes";
 const char* kMasterNodesPropName = "master_nodes";
-const char* kHostToOwnedGlobalIDsPropName = "host_to_owned_global_ids";
+const char* kHostToOwnedGlobalNodeIDsPropName = "host_to_owned_global_node_ids";
```

Comment:
This will break backward compatibility. Please treat it the same way kDeprecatedLocalToGlobalIDPropName is treated.

Reply:
Good catch